docker-compose.yml

```diff
@@ -10,3 +10,5 @@ services:
       - ./src:/usr/app/src
       - ./logs:/usr/app/logs
       - ./db:/usr/app/db
+    logging:
+      driver: gcplogs
```

Comment:
I think this is for running docker containers locally, so we prob don't want to use the gcp driver here

Reply:
okay yeah I'll remove it from local docker-compose